robot_smach_states/examples/util/designators/example_attrDesignator.py

```diff
@@ -0,0 +1,35 @@
+#ROS
```

Comment:
This filename ain't right, why the capital?

Reply:
Changed it to example_attribute_designator.py now.